pyupgrade.py

```diff
@@ -1022,6 +1022,7 @@ def __init__(self):
 
         self._class_info_stack = []
         self._in_comp = 0
+        self._in_async_def = False
```

Comment:
move this down one line so that the state is associated with the thing it needs (for instance `_in_comp` is affecting `super_calls`)